blocks/library/gallery/block.js

```diff
@@ -127,6 +127,15 @@ class GalleryBlock extends Component {
 		);
 	}
 
+	//deselect images when losing focus
```

Comment:
As a function, I think it might be more appropriate here to use DocBlock syntax:

```
/**
 * Deselect images when losing focus
 */
```

...but then again, there's nothing inherently about `componentWillReceiveProps` that is related to losing focus, so I'd suggest either moving the comment to be an inline comment, or create a separate standalone function specific to the deselect behavior.